### DISCUSSION TITLE
What minimalistic configuration and where is default config file?

### DISCUSSION THREAD

**Discussion by minsoehan** [hyprwm/Hyprland#8569] [General]
I find myself pleased with default configuration of Hyprland. But I don't find default configuration file. The file `/usr/share/hypr/hyprland.conf` file is an example, not default, especially animation. So I created my `~/.config/hypr/hyprland.conf` file from scratch with the only lines I really need. I wonder if it gives any performance benefits if I have the config file with the only lines as few as I need? And where is the default file? I know I can find default values in the tables on Hyprland wiki page. But I want to know if there is default config file in my machine.

**Comment by fufexan** [comment]
There is no default config. If Hyprland cannot find one when starting (in XDG_CONFIG_DIRS or XDG_CONFIG_HOME), it will create one in XDG_CONFIG_HOME. The default values can be found in https://github.com/hyprwm/Hyprland/blob/0ddb952d7a3d102e160eea93c68f0d366186cf73/src/config/ConfigManager.cpp#L329.
